pypeit/scripts/skysub_regions.py

```diff
@@ -55,7 +57,8 @@ def main(args):
             detname = DetectorContainer.get_name(det)
 
         # Load it up
-        spec2DObj = spec2dobj.Spec2DObj.from_file(args.file, detname, chk_version=True)
+        spec2DObj = spec2dobj.Spec2DObj.from_file(args.file, detname,
+                                                  chk_version=(not args.try_old))
```

Comment:
In quite a few places, there's `(not args.try_old)`. Would it be cleaner to define a variable `chk_version=not args.try_old`, and then replace all instances with `chk_version=chk_version`? Probably just a difference of style...

Reply:
Good recommendation.  All the scripts using `try_old` now define `chk_version = not args.try_old` somewhere near the top of the script.